### PR TITLE
[9522] Add workflow to automate SDK version bumps

### PR DESCRIPTION
**Pull request by andlum**
## Summary
Add Github Action that automates:
- updating `build.gradle` for Android SDK version bumps
- updating `Cartfile.resolved` + `react-native-radar.podspec` for iOS SDK version bumps
- updating `package.json` and `package-lock.json`

### CODE REVIEW

File: .github/workflows/update-native-sdks.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Bump Native SDK version numbers whenever a repository dispatch of a release
+# is received
+name: Bump Native SDKs
+on:
+  repository_dispatch:
+    types: ['ios-sdk-release', 'android-sdk-release']
+jobs:
+  update_android_sdk_version:
+    runs-on: ubuntu-latest
+    if: github.event.client_payload.platform == "android"
```
Comment (tjulien) [inline .github/workflows/update-native-sdks.yml:10]: I think android sdk repo might be missing this right now

Comment (tjulien) [reply .github/workflows/update-native-sdks.yml:10]: `client-payload` (not underscore), right?

File: .github/workflows/update-native-sdks.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      # open a pull request with the new sdk version
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          title: Automated radar-sdk-android version bump to ${{ github.event.client_payload.release }}
+          reviewers: radarlabs/eng
+          token: ${{ secrets.GITHUB_TOKEN }}
+  update_ios_sdk_version:
+    runs-on: ubuntu-latest
+    if: github.event.client_payload.platform == 'iOS'
```
Comment (tjulien) [inline .github/workflows/update-native-sdks.yml:45]: `client-payload` (not underscore), right?

Also, `ios` not `iOS`

File: .github/workflows/update-native-sdks.yml (hunk 1 above)
Comment (andlum) [reply .github/workflows/update-native-sdks.yml:10]: It is underscore. It's one of those annoying instances where the workflow we're using uses hyphens, but the resultant payload is underscores: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#repository_dispatch

Comment (tjulien) [reply .github/workflows/update-native-sdks.yml:10]: annoying

**Review by tjulien — APPROVED** [review]

**Pull request merged**